MIGRATION_TO_2.1.md

```diff
@@ -26,11 +26,14 @@ The maven group IDs and package names have changed as follows:
 Interface changes
 ----
 
-The base java package for the `FluxCapacitor`, `Workflow`, `WorkflowStep`, and other generic interfaces and annotations has been changed, and any that weren't in the `flux-common` maven package have been moved there:
+The base java package for `FluxCapacitor`, `Workflow`, `WorkflowStep`, and other generic interfaces, annotations, and exceptions has been changed, and any that weren't in the `flux-common` maven package have been moved there:
```

Comment:
nit: extra comma after annotations

Reply:
I like the [oxford comma](https://en.wikipedia.org/wiki/Serial_comma) ;)